packages/app/src/controller/staking/metadata.ts

```diff
@@ -35,7 +35,7 @@ export async function GetHeikoStakingMetadataSnapshots(ctx: Context, next: Next)
     take: pageSize,
     skip
   })
-  const { blockHeight } = list[0] // get the latest block height
+  const { blockHeight } = list[0] ? list[0] : 0 // get the latest block height: ;
```

Comment:
```suggestion
  const { blockHeight } = list?.[0] || 0
```
Is there any calculation impact if we give a default 0?